checkov/terraform/checks/resource/aws/BackupVaultEncrypted.py

```diff
@@ -0,0 +1,20 @@
+from checkov.common.models.enums import CheckCategories
+from checkov.terraform.checks.resource.base_resource_value_check import BaseResourceValueCheck
+from checkov.common.models.consts import ANY_VALUE
+
+
+class BackupVaultEncrypted(BaseResourceValueCheck):
+    def __init__(self):
+        name = "Ensure Backup Vault server-side encryption is enabled."
+        id = "CKV_AWS_165"
+        supported_resources = ['aws_backup_vault']
+        categories = [CheckCategories.BACKUP_AND_RECOVERY]
```

Comment:
the category should be different 😉 
```suggestion
        categories = [CheckCategories.ENCRYPTION]
```